src/pages/home/report/ReportActionCompose/ReportActionCompose.tsx

```diff
@@ -226,8 +229,9 @@ function ReportActionCompose({
             !isAdminRoom(report) &&
             !isConciergeChatReport(report) &&
             !isInvoiceReport(report) &&
-            !isGroupChat(report),
-        [report],
+            !isGroupChat(report) &&
+            !isSettled(parentReport),
```

Comment:
```suggestion
            !isSettled(parentReport) &&
            !isSettled(report),
```
Noticed it doesn't work for expenses.

Reply:
@getusha why it should work for expenses? How can you have paid expense?

Reply:
@getusha can you show me (recording) of a case that it's not working but should?

Reply:
or share repro steps?

Reply:
>or share repro steps?

1. Submit an expense to a workspace
2. Pay the expense
3. Navigate to Reports > Open the expense you paid
4. Now drag a file

here is a video: https://github.com/Expensify/App/pull/63834#discussion_r2149910988

Reply:
@getusha thanks! I still cannot recreate this - can you share the steps to get this type of expense?